src/miniscript/types/extra_props.rs

```diff
@@ -455,6 +506,10 @@ impl Property for ExtData {
             stack_elem_count_dissat: self.stack_elem_count_dissat.map(|x| x + 1),
             max_sat_size: self.max_sat_size.map(|(w, s)| (w + 2, s + 1)),
             max_dissat_size: self.max_dissat_size.map(|(w, s)| (w + 1, s + 1)),
+            // TODO: fix dissat stack elem counting above in a later commit
+            // Technically max(1, self.exec_stack_elem_count_sat), same rationale as cast_dupif
```

Comment:
I'm curious why you defer this to a later commit. You have `cmp::max` :)

Reply:
That is a bug fix in existing code about `max_stack_elem_count` (different from exec count) #268 which should have a separate commit. Opened #268 for it